app/scripts/keycloak-init.js

```diff
@@ -26,39 +29,61 @@
     }).error(function () {
       window.location.reload();
     });
+  }
 
-  });
+  function configureAuth() {
+    app.factory('Auth', function () {
+      return auth;
+    });
 
-  app.factory('Auth', function () {
-    return auth;
-  });
+    app.factory('authInterceptor', function ($q, Auth) {
+      return {
+        request: function (config) {
+          var deferred = $q.defer();
+
+          if (config.url === 'rest/sender' || config.url === 'rest/registry/device/importer') {
```

Comment:
let's add a comment, that these are never protected by the plugged-in AUTH library.

They are covered by custom HTTP_Basic fu